gradle/dependencies.gradle

```diff
@@ -34,13 +34,13 @@ versions += [
   junit: "4.12",
   lz4: "1.3.0",
   metrics: "2.2.0",
-  powermock: "1.6.3",
+  powermock: "1.6.4",
   reflections: "0.9.10",
   rocksDB: "3.10.1",
```

Comment:
I noticed that too, but I don't know why. @guozhangwang is it intentional that streams is using RocksDB 3.x instead of 4.x?